dmutils/framework_notification.py

```diff
@@ -0,0 +1,8 @@
+from datetime import date
+
+# Go Live Dates
+DOS6_GO_LIVE_DATE = '2022-01-14'
+
+
+def dos6_live(params) -> bool:
```

Comment:
I think you raise a good point, I think I will change the function name to new_frameworks_live. As you say, I don't think we'll need to update this logic but we will probably need to update the text so making this method framework agnostic seems sensible.

Reply:
👍 

Just a minor suggestion. I tend to prefer functions that start with a verb because they make it obvious what the function is doing (e.g. we're checking if we should show the new frameworks banner, we're not making the new frameworks live). `are_new_frameworks_live`?

Reply:
I will change the method to `are_new_frameworks_live`. This one of the things I like about Ruby as I can put a `?` at the end of any methods that return booleans, makes it nicer to read. Alas, I don't think this has caught on other languages.